autoscheduler/scraper/management/commands/scrape_depts.py

```diff
@@ -0,0 +1,27 @@
+# Called to execute the beginning of scaping courses
+from django.core.management import base
+from scraper.banner_requests import BannerRequests
+from scraper.models import Department
+
+import time
+import datetime
+
+class Command(base.BaseCommand):
+    """ Gets all departments from banner and adds them to the database """
+
+    def scrape_departments():
+        term = add_arguments()
+        var request = BannerRequests(term)
+        depts = request.get_departments(term,amount)
+
+    def add_arguments(self, parser):
+        parser.add_argument('--term', type=str, required=False, default="201931")
+
+    def handle(self, *args, **options):
+        start = time.time()
+        scrape_departments()

```

Comment:
To actually access the term from `scrape_departments`, you need to change it to accept a `term: str` argument, and then change this line in handle to 
```suggestion
        scrape_departments(options['--term'])
```